### PR TITLE
New features with Datadog instrumentation

### PR DESCRIPTION
**Pull request by ib-steffen**

### CODE REVIEW

File: cmd/chaosmania/otlp.go [hunk 1]
```diff
@@ -3,13 +3,14 @@ package main
...
+	"strings"
 
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
 	"go.opentelemetry.io/otel/propagation"
 	"go.opentelemetry.io/otel/sdk/resource"
-	sdktrace "go.opentelemetry.io/otel/sdk/trace"
-	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
+	"go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
```
Comment (esara) [inline cmd/chaosmania/otlp.go]: why downgrade? the current semconv is going from 21 to 24?

**Review by enlinxu — APPROVED** [review]

File: helm/video/values.yaml [hunk 2]
```diff
@@ -167,21 +176,24 @@ services:
   - name: "data-broker-producer"
     type: kafka-producer
     config:
+      tracing_service_name: data-broker
       brokers: 
-        - # Required
-      username: # Required
-      password: # Required
+        - ehtestcausely.servicebus.windows.net:9093
```
Comment (esara) [inline helm/video/values.yaml]: maybe leave this empty, if we want other people to try?

File: .DS_Store
Comment (esara) [inline .DS_Store:1]: nit drop this from the commit

File: cmd/chaosmania/server.go [hunk 3]
```diff
@@ -22,89 +23,74 @@ import (
...
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		fmt.Fprintf(w, "Error: Method not allowed")
+	}
+}
+
+func handleHealth(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+}
 
+func runWitheOTEL(log *zap.Logger, port int64) {
```
Comment (esara) [inline cmd/chaosmania/server.go]: what is eOTEL?

File: helm/boutique/templates/_helpers.tpl [hunk 4]
```diff
@@ -51,6 +96,9 @@
     httpGet:
       path: /health
       port: http
+  volumeMounts:
+    - name: apmsocketpath
+      mountPath: /var/run/datadog
```
Comment (esara) [inline helm/boutique/templates/_helpers.tpl:102]: I am wondering the pros/cons of

https://docs.datadoghq.com/containers/kubernetes/apm/?tab=unixdomainsocketuds
vs
https://docs.datadoghq.com/containers/kubernetes/apm/?tab=tcp

ps: I used the latter in my configuration

File: helm/video/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: boutique
```
Comment (esara) [inline helm/video/Chart.yaml]: video?

**Pull request merged**